src/github_runner_image_builder/openstack_builder.py

```diff
@@ -248,13 +249,37 @@ def run(
         _wait_for_cloud_init_complete(conn=conn, server=builder, ssh_key=BUILDER_KEY_PATH)
         image = store.create_snapshot(
             cloud_name=cloud_config.cloud_name,
-            image_name=IMAGE_SNAPSHOT_NAME,
+            image_name=image_config.name,
             server=builder,
             keep_revisions=keep_revisions,
         )
         logger.info("Requested snapshot, waiting for snapshot to complete: %s.", image.id)
         _wait_for_snapshot_complete(conn=conn, image=image)
+        if (
+            cloud_config.upload_cloud_name
+            # and cloud_config.cloud_name != cloud_config.upload_cloud_name
```

Comment:
should this outcommented line stay as it is?

Reply:
Updated, thank you for the review!